include/cctz/civil_time_detail.h

```diff
@@ -566,6 +567,198 @@ CONSTEXPR_F civil_day prev_weekday(civil_day cd, weekday wd) noexcept {
   }
 }
 
+////////////////////////////////////////////////////////////////////////
+
+enum class weekstart
+{
+  monday_zero,
+  monday_one,
+  sunday_zero,
+  sunday_one,
+};
+
+CONSTEXPR_F int weekday_to_number(weekday wd, weekstart ws) noexcept {
```

Comment:
weekstart could basically always be static. So I wonder if we can use template
so this would be

weekday_to_number<weekstart::xxx>();
or even
weekday_to_number<weekstart::xxx, int>();

but the way I now implemented it seems to be most aligned with cctz
current api.